src/description.cpp

```diff
@@ -1051,6 +1054,17 @@ string Description::Media::generateSdpLines(string_view eol) const {
 	return sdp.str();
 }
 
+void Description::Media::parseFirstLine(string line) {
+  std::istringstream ss(line);
+  string word;
+  std::getline(ss, word, ' '); // audio
+  std::getline(ss, word, ' '); // 9
+  std::getline(ss, word, ' '); // UDP/TLS/RTP/SAVPF
```

Comment:
The logic here is different from the one in `Entry()`, meaning the behavior can be inconsistent, for instance if there are two consecutive spaces.